src/v2/Apps/Home/Components/HomeHeroUnits/HomeHeroUnit.tsx

```diff
@@ -276,10 +276,9 @@ export const HomeHeroUnitFragmentContainer = createFragmentContainer(
 )
 
 export const LOGGED_OUT_HERO_UNIT: StaticHeroUnit = {
-  title: "Collect art from leading galleries, fairs, and auctions",
+  title: "Collect art by the world's leading artists",
   subtitle: "Sign up to get updates about your favorite artists",
   href: "/signup",
   linkText: "Sign up",
-  backgroundImageURL:
-    "https://files.artsy.net/images/alexander-calder-rouge-triomphant-triumphant-red-1959-1965.jpg",
+  backgroundImageURL: "http://files.artsy.net/homepage/signup-banner.png",
```

Comment:
Should this be a `.png`? We should be able to get a much lower file-size if it were a jpg.

Reply:
Gemini will convert it here: https://github.com/artsy/force/blob/d6f7d907c5db1b1f0ae2c9ea09225331513db800/src/v2/Apps/Home/Components/HomeHeroUnits/HomeHeroUnit.tsx#L61-L68

Reply:
Gotcha, I'll change the file extension in vanity

Reply:
You'd have to re-encode it (in Photoshop for instance) rather than change it's file extension. But this isn't necessary, Gemini (our image processing service), will re-encode this to JPG (or WebP) automatically.

Reply:
Wait, does just changing the file extension change the file type?

Reply:
If not, I might have to petition to design to get a .jpg which isn't a big deal.

Reply:
I believe what @dzucconi is saying is that all is ok -- that `cropped` function up above will take care of everything automatically behind the scenes (read png -> user gets served a jpg).

Reply:
Gotcha, I was going to mention that there are other imgs in vanity that are .pngs but it looks like Gemini does the file conversion behind the scenes so folks don't have to worry about that.